weldr-client.spec.in

```diff
@@ -23,6 +23,8 @@ Source2:   https://keys.openpgp.org/vks/v1/by-fingerprint/%%GPGKEY%%#/gpg-%%GPGK
 Obsoletes: composer-cli < 35.0
 Provides: composer-cli = %{version}-%{release}
 
+Requires: diffutils
```

Comment:
Maybe `Suggests`? Most of `weldr-client` will work even without diffutils. And the `composer-cli diff` command can handle missing `diff` gracefully.

Reply:
If it needs it to function completely I think it should require it.